_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+  other recent PSBT-related GUI changes (see Newsletters [#74][news74
+  psbtgui] and [#82][news82 psbtgui]) and [HWI's own GUI][hwi gui] this
+  makes it possible to use a PSBT-based process with Bitcoin Core for
+  the first time without using any RPCs.
+
+    ![Screenshot of PSBT dialog in Bitcoin Core GUI](/img/posts/2020-06-psbt-dialog.png)
+
+- [Bitcoin Core #16377][] updates the `walletcreatefundedpsbt` and
+  `fundrawtransaction` RPCs.  These RPCs normally use the wallet to
+  automatically choose which UTXOs to spend in a unsigned transaction,
+  but they also allow the user to specify one or more UTXOs they want to
+  spend in that transaction.  Previously, if the UTXOs selected by the
+  user weren't enough to pay for all of the transaction's outputs, the
+  wallet would automatically select more UTXOs to spend.  But, if a
+  user is manually selecting UTXOs, they may have some reason they don't
+  want to spend additional UTXOs, the the RPCs will now fail by default
+  if the user manually selects any UTXOs.  This default may be overridden
+  using the new `add_inputs` parameter to both RPCs.
+
+- [Eclair #1461][] adds several API endpoints that forward to Bitcoin
+  Core RPCs for getting that program's wallet balances and other
+  information.  The goal is to make it easier to integrate Eclair with
+  the [Ride The Lightning][] node management dashboard.
+
+- [Bitcoin Core #19071][] adds documentation describing how developers
+  can contribute to the new and experimental [Bitcoin Core GUI
+  repository][].  Pull requests related to the GUI should be made into
+  this new repository, which will be bidirectionally synced with the
+  [main repository's master development branch][bitcoin/bitcoin] using
+  the *monotree* development model used by the Linux Kernel Project.
+  There are no direct user-visible changes from this split---users will
+  still receive the GUI in official packaged versions of Bitcoin Core or
+  when building using `--with-gui` from the source code in the main
+  repository.
+
+    FIXME:harding illustration of monotree vs monorepo
+
+    This split is an experiment designed to determine whether using
+    different repositories for different subsystems will help people
+    interested in a particular subsystem focus on that part of the
+    project.  For example, someone using the GitHub *Watch Repository*
+    feature will receive fewer issue and PR status updates each day when
+    watching the Bitcoin Core GUI repository rather than the main
+    repository, making it easier for them to monitor the project.
+    Conversely, developers watching the main project, who may not all
+    be interested in the GUI, will no longer need to receive
+    notifications about it.  In the best case, it's hoped that this
+    improved focus can speed up development---allowing the the project to
+    create monotree repositories for other parts of the project.
```

Comment:
The dash makes it seem like "allowing the project to create monotree repositories for other parts of the project." is the goal. I think it should be a separate sentence to clarify that separating the project further is possible but isn't proposed right now.

Also, s/the the/the/